pandas/tests/window/test_base_indexer.py

```diff
@@ -166,3 +172,8 @@ def test_rolling_forward_window(constructor, func, np_func, expected, np_kwargs)
     tm.assert_equal(result, expected)
     expected2 = constructor(rolling.apply(lambda x: np_func(x, **np_kwargs)))
     tm.assert_equal(result, expected2)
+    # Check that the function works without min_periods supplied
```

Comment:
can you put a blank line before each case

and add comments for lines 169, 173.

Reply:
@jreback 
Done, all green.